packages/Popover/src/positionUtils.spec.js

```diff
@@ -0,0 +1,274 @@
+import { Align, Justify } from './Popover';
+import { calculatePosition, getElementPosition } from './positionUtils';
+
+const SPACING = 10;
+const CENTERED_REFELPOS = {
+  bottom: 393.3948802947998,
+  height: 19,
+  left: 482.585205078125,
+  right: 541.0511131286621,
+  top: 374.78692626953125,
+  width: 58,
+};
+const TOP_LEFT_REFELPOS = {
+  bottom: 18.607954025268555,
+  height: 19,
+  left: 0,
+  right: 58.46590805053711,
+  top: 0,
+  width: 58,
+};
+const TOP_RIGHT_REFELPOS = {
+  bottom: 18.607954025268555,
+  height: 19,
+  left: 965.17041015625,
+  right: 1023.6363182067871,
+  top: 0,
+  width: 58,
+};
+const BOTTOM_LEFT_REFELPOS = {
+  bottom: 768.181806564331,
+  height: 18,
+  left: 0,
+  right: 58.46590805053711,
+  top: 749.5738525390625,
+  width: 58,
+};
+const CONTENTELPOS = {
+  bottom: 48.8636360168457,
+  height: 61,
+  left: 0,
+  right: 123.21591186523438,
+  top: 0,
+  width: 154,
+};
+global.window.innerWidth = 1024;
+global.window.innerHeight = 768;
+
+describe('unit tests', () => {
+  describe('calculatePosition', () => {
+    test('TOP: it aligns content to top, when the space is available and the prop is set', () => {
+      const pos = calculatePosition({
```

Comment:
Can we add an explicit type response to calculatePosition? I forgot to include that, and it'd be good to have that be explicitly typed as we do rely on it heavily.

Reply:
working on doing this now